msg/json/en.json

```diff
@@ -18,6 +18,7 @@
 	"DELETE_X_BLOCKS": "Delete %1 Blocks",
 	"DELETE_ALL_BLOCKS": "Delete all %1 blocks?",
 	"CLEAN_UP": "Clean up Blocks",
+	"CLOSE": "Close",
```

Comment:
After further investigation, I think you need to put this in msg/messages.js instead.

Reply:
Done.